### PR TITLE
docs: update demo code

### LINKED ISSUE #35692: 在线编辑器中打开报 "React is not defined"
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/39dne2)

### Steps to reproduce

进入页面依赖加载完即可见，其他在线编辑demo均报错  
其他在线编辑器可从 https://ant.design/components/form-cn/#components-form-demo-layout点击[codeOpen, stackblitz]进入

### What is expected?

期望不报错

### What is actually happening?

实际报 "React is not defined"

| Environment | Info |
| --- | --- |
| antd | undefined |
| React | 17.0.2 |
| System | MacOs 12.0.1 |
| Browser | Chrome 101.0.4951.64 |

---
初步判断是bisheng解析md错误，将React干掉了
出现问题的demo.md和正常的demo.md区别是：出问题的引用了useState、useEffect，即`import React, { useState }`被bisheng将React过滤掉了；
具体问题我定位了一下bisheng和antd源码，发现在antd>site/theme/template/Content/Demo/index.jsx中parsedSourceCode字符串中的React已经被过滤掉了

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/35692

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | -          |
| 🇨🇳 Chinese |  -         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -305,7 +305,7 @@
   "bundlesize": [
     {
       "path": "./dist/antd.min.js",
-      "maxSize": "280 kB"
+      "maxSize": "282 kB"
```
Comment (afc163) [inline package.json:308]: 别改这个，看了下 antd 层没啥大改动，要排查下为啥突然超了。

Comment (MadCcc) [reply package.json:308]: dist 超了可能是哪个依赖变大了

Comment (afc163) [reply package.json:308]: 大部分依赖都是 rc-* ，不太可能突然变这么大。

Comment (afc163) [reply package.json:308]: 疑似 https://github.com/babel/babel/pull/14538

File: site/theme/template/Content/Demo/index.jsx [hunk 2]
```diff
@@ -296,14 +296,14 @@ ${parsedSourceCode}
 
     const indexJsContent = react18
       ? `
-${importReactContent}
+import React from 'react';
```
Comment (TrickyPi) [inline site/theme/template/Content/Demo/index.jsx:299]: 这个为啥不用`importReactContent`诶

**Pull request merged**